lib/mimdb_web/templates/genre/show.html.eex

```diff
@@ -0,0 +1,13 @@
+<h1>Show Genre</h1>
```

Comment:
It would be ok to remove this file (and all show related code) if you don't think it will be used. Your code has removed the need for it...so it is safe to remove unless there are mitigating circumstances.

Reply:
I'm pondering....  There's a test for :delete, that verifies that the deleted genre is indeed gone, by trying to :show it.

I could replace the

Reply:
Yes, great catch. You could update that test to refute existence of that Genre on Index page. Your call, I don't think this is a critical change, but I _do_ like less code. Less code means less things to break, :)

Reply:
I've added a "refute" test as suggested